slackbridge/utils.py

```diff
@@ -182,7 +182,22 @@ def format_slack_message(text):
 
     TODO: Preserve bold and italics (convert to markdown?)
     """
-    return re.sub(r'\x03(?:\d{1,2}(?:,\d{1,2})?)?', '', text, flags=re.UNICODE)
+
+    def nick_replace(match):
+        """
+        Replace any IRC nick of the form keur to <@keur> if it exists in
+        the provided list of Slack display names. To prevent accidental
+        conversions, "no-more-slack" will not be converted to "<@no-more>",
+        assuming no user has the display name "no-more" in the Workspace.
+        """
+        nick = match.group(1)
+        if nick in users:
+            return '<@{}>'.format(nick)
+
+    text = re.sub(r'\x03(?:\d{1,2}(?:,\d{1,2})?)?', '', text, flags=re.UNICODE)
+    # we can be greedy here; nick is checked against valid list of users
+    text = re.sub(r'([^\s]+)-slack', nick_replace, text)
```

Comment:
This is fine for now, but I think we should make the suffix `-slack` into a constant somewhere instead of repeating it several times throughout this repo.